salt/modules/virt.py

```diff
@@ -2724,3 +2866,222 @@ def pool_define_build(name, **kwargs):
         return (True, 'Pool exist')
 
     return True
+
+
+def list_pools(**kwargs):
+    '''
+    List all storage pools.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.list_pools
+    '''
+    conn = __get_conn(**kwargs)
+    pools = [pool.name() for pool in conn.listAllStoragePools()]
+    conn.close()
+    return pools
+
+
+def pool_info(name, **kwargs):
+    '''
+    Return informations on a storage pool provided its name.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_info default
+    '''
+    result = {}
+    conn = __get_conn(**kwargs)
+    try:
+        pool = conn.storagePoolLookupByName(name)
+        infos = pool.info()
+        states = ['inactive', 'building', 'running', 'degraded', 'inaccessible']
+        state = states[infos[0]] if infos[0] < len(states) else 'unknown'
+        result = {
+            'uuid': pool.UUIDString(),
+            'state': state,
+            'capacity': infos[1],
+            'allocation': infos[2],
+            'free': infos[3],
+            'autostart': pool.autostart(),
+            'persistent': pool.isPersistent()
+        }
+    finally:
+        conn.close()
+        return result  # pylint: disable=lost-exception
+
+
+def pool_start(name, **kwargs):
+    '''
+    Start a defined libvirt storage pool.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_start default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.create() == 0
```

Comment:
You use this `foo == 0` all over the place. Unless `0` means something (like a constant, say error code or so) then it has to be aligned to it. Otherwise I would use just `ret = bool(pool.create())`.